src/modules/projects/components/tables/ProjectReferralHouseholdMembersTable.tsx

```diff
@@ -0,0 +1,52 @@
+import GenericTable, { ColumnDef } from '@/components/elements/GenericTable';
+import ClientName from '@/modules/client/components/ClientName';
+import HohIndicator from '@/modules/hmis/components/HohIndicator';
+import { parseAndFormatDate } from '@/modules/hmis/hmisUtil';
+import { HmisEnums } from '@/types/gqlEnums';
+import { ReferralPostingDetailFieldsFragment } from '@/types/gqlTypes';
+
+type Row = ReferralPostingDetailFieldsFragment['householdMembers'][number];
+
+const columns: ColumnDef<Row>[] = [
+  {
+    header: '',
+    key: 'indicator',
+    width: '1%',
+    render: (row: Row) => (
+      <HohIndicator relationshipToHoh={row.relationshipToHoH} />
+    ),
+  },
+  {
+    header: 'Member Name',
+    render: ({ client }: Row) => (
+      <ClientName
+        client={client}
+        linkToProfile={client.access.canViewEnrollmentDetails}
+      />
+    ),
+  },
+  {
+    header: 'Relationship to HoH',
+    render: (row: Row) => HmisEnums.RelationshipToHoH[row.relationshipToHoH],
```

Comment:
Same here, we could use the `HmisEnum` component. Not a huge deal, the main benefit is that the `INVALID` enum would be displayed in red. We have a system whereby numeric values that can't be mapped to the gql enum are resolved as INVALID .